packages/static-wado-webserver/lib/dicomWebServerConfig.mjs

```diff
@@ -10,11 +12,23 @@ const { staticWadoConfig } = StaticCreator;
 const { dicomWebServerConfig } = ConfigPoint.register({
   dicomWebServerConfig: {
     configBase: staticWadoConfig,
+    dicomWebScpConfig,
```

Comment:
this is not the right way to include the configuration.  The ae configuration is referenced by name, and gets defined in staticWadoConfig, and then references something in the aeConfig.  You can get access to aeConfig from @ohif/static-wado-util.   Then, you just reference it in the staticWadoConfig or over-ride it in dicomWebServerConfig, for example:
    queryAe: "dcmqrscp",
inside static-wado-remote.json5  and then you use that configuration file - either by having as your default configuration file, or by using -c tests/static-wado-remote.json5.

Reply:
Yes, correct. I decided to let as is (prior to PR) to hear from you the best approach considering the whole stuff about configuration.